people/service.go

```diff
@@ -100,46 +136,49 @@ func (s *peopleServiceImpl) getPersonByUUID(uuid string) (person, bool, error) {
 		return person{}, false, err
 	}
 	if len(cachedValue) == 0 {
-		log.Infof("INFO No cached value for [%v]", uuid)
+		log.Infof("INFO No cached value for [%v].", uuid)
 		return person{}, false, nil
 	}
 	var cachedPerson person
 	err = json.Unmarshal(cachedValue, &cachedPerson)
 	if err != nil {
-		log.Errorf("ERROR unmarshalling cached value for [%v]: %v", uuid, err.Error())
+		log.Errorf("ERROR unmarshalling cached value for [%v]: %v.", uuid, err.Error())
 		return person{}, true, err
 	}
 	return cachedPerson, true, nil
 
 }
 
 func (s *peopleServiceImpl) openDB() error {
+	s.Lock()
```

Comment:
The defer is now in this function so more obvious about what unlocks it. Tested it without the lock but figured no harm in keeping it there.